data/json/regional_map_settings.json

```diff
@@ -916,7 +916,51 @@
       "base_acid": 0.0,
       "base_wind": 3.4,
       "base_wind_distrib_peaks": 80,
-      "base_wind_season_variation": 50
+      "base_wind_season_variation": 50,
+      "mist": {
+        "mist_active": true,
+        "mist_scaling": 1.0,
+        "mist_frequency": 7,
+        "mist_length": 8,
```

Comment:
Duration should be randomized between a `[ min, max ]` pair rather than fixed randomization around a value.